### PR TITLE
[3.5e] Tome of Magic: Some minor corrections

### PR DESCRIPTION
**Pull request by jagbear**
Changes to fix deprecated issues

### CODE REVIEW

File: data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst [hunk 1]
```diff
@@ -227,6 +227,10 @@ Multiclass Note		KEY:Witch Slayer ~ Multiclass Note			CATEGORY:Special Ability	T
...
+
+Cloak of Shadows		KEY:Child of Night ~ Cloak of Shadows			CATEGORY:Special Ability	TYPE:Child of Night Class Feature.SpecialQuality.Supernatural	SOURCEPAGE:p.117	DESC:Your fl esh takes on a hazy, almost translucent hue, granting a bonus equal to your child of night level on Hide checks. At 4th level, once per day as a swift action, you can make your outline shadowy and indistinct, as the blur spell, for a number of minutes equal to your class level. When you reach 6th level, you can use the blur effect three times per day. At 8th level, your form permanently becomes hazy and you continually have a blur effect. At 10th level, two times per day as a swift action, you can make your true location more diffi cult to discern, as if you were affected by the displacement spell, for 10 rounds.
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst]: Why the duplicate ability?

File: data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst [hunk 2]
```diff
@@ -237,7 +241,10 @@ Night Form			KEY:Child of Night ~ Night Form				CATEGORY:Special Ability	TYPE:Ch
...
+
+
+Mysteries/Spellcasting	KEY:Master of Shadow ~ Mysteries/Spellcasting		CATEGORY:Special Ability	TYPE:Master of Shadow Class Feature.SpecialQuality	SOURCEPAGE:p.122	DESC:At 2nd level and every level thereafter, you gain new mysteries or spells per day and an increase in caster level (and mysteries or spells known, if applicable) as if you had also gained a level in a casting class to which you belonged before adding the prestige class level. You do not, however, gain any other benefi ts a character of that class would have gained. If you had more than one casting class before becoming a master of shadow, you must decide to which class to add each level for the purpose of determining spells per day, caster level, and spells or mysteries known. 
+Rapid Shadow	KEY:Master of Shadow ~ Rapid Shadow	CATEGORY:Special Ability	TYPE:Master of Shadow Class Feature.SpecialQuality.Supernatural	SOURCEPAGE:p.	DESC:After you reach 10th level, if your shadow servant dies you can summon a replacement 10 minutes later.
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst]: Good catch

File: data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst [hunk 3]
```diff
@@ -262,7 +269,10 @@ Far Shadow			KEY:Unseen Weapon ~ Far Shadow			CATEGORY:Special Ability	TYPE:Shad
...
+
 Touch of Shadow			KEY:Shadowsmith ~ Touch of Shadow				CATEGORY:Special Ability	TYPE:Shadowsmith Class Feature.SpecialQuality.Supernatural	SOURCEPAGE:p.133	DESC:At 1st level, your ability to manipulate shadow is undeveloped. You cannot yet create actual items, but you can surround your hands and feet in a layer of shadow. This ability grants you a bonus on Climb checks equal to your class level. Further, it distributes your weight more evenly, so you cannot be tracked. Touch of shadow lasts for 1 minute per class level and is usable a number of times per day equal to your class level.
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst]: Why the empty spaces here?

**Comment by BahamutDragon** [comment]
Mate, 

Happy Christmas. 

Not seeing any deprecated issues, which is a bit misleading for the title. I see a fix to the class skills, an ability that got tucked up on the wrong line and a duplicate class ability that you added. Not sure what the intention is behind adding a few extraneous blank lines.  

A quick word on terminology, a deprecated issue would be taking something like AUTO:FEAT, which is deprecated in the 6.06 cycle, and change it to ABILITY:FEAT|AUTOMATIC.

I can easily accept this as is, no worries there. In the future, might be best to commit these changes to a separate branch. It is customary for the person submitting the PR to correct issues found during review, but as it is Christmas and your first PR that has actual content, I will handle the minor fixes.

Best,

**Comment by jagbear** [comment]
Merry Christmas.
The last submission had to do with variables defined with a non-zero
amount, which the error message on loading said was deprecated.  I don't
know how things got duplicated.  It happened before as well with the minor
tome of magic changes I submitted.  I will be happy to make any corrections
necessary.


On Fri, Dec 25, 2015 at 12:00 PM, Bahamut <notifications@github.com> wrote:

> Mate,
>
> Happy Christmas.
>
> Not seeing any deprecated issues, which is a bit misleading for the title.
> I see a fix to the class skills, an ability that got tucked up on the wrong
> line and a duplicate class ability that you added. Not sure what the
> intention is behind adding a few extraneous blank lines.
>
> A quick word on terminology, a deprecated issue would be taking something
> like AUTO:FEAT, which is deprecated in the 6.06 cycle, and change it to
> ABILITY:FEAT|AUTOMATIC.
>
> I can easily accept this as is, no worries there. In the future, might be
> best to commit these changes to a separate branch. It is customary for the
> person submitting the PR to correct issues found during review, but as it
> is Christmas and your first PR that has actual content, I will handle the
> minor fixes.
>
> Best,
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BahamutDragon/pcgen/pull/344#issuecomment-167254589>.
>



-- 

   JD


  ("`-''-/").___..--''"`-._
   `6_ 6  )   `-.  (     ).`-.__.`)
   (_Y_.)'  ._   )  `._ `. ``-..-'
 _..`--'_..-_/  /--'_.' ,'
(il),-''  (li),'  ((!.-'

**Comment by BahamutDragon** [comment]
Changes already pushed in to correct the items. Even added a few bonuses.

I just reviewed the change log thoroughly, nothing dealing with DEFINEs in this PR, just a few items in desc corrections and mirror alterations to the Tome of Magic.

**Pull request merged**